ENGAGEHF/Questionnaire/QuestionnaireSheetView.swift

```diff
@@ -40,37 +43,43 @@ struct QuestionnaireSheetView: View {
                     }
 #endif
                     
-                    do {
-                        try await standard.add(response: questionnaireResponse)
-                        dismiss()
-                    } catch {
-                        viewState = .error(AnyLocalizedError(error: error))
+                    Task {
+                        do {
+                            try await standard.add(response: questionnaireResponse)
+                            try? await Task.sleep(nanoseconds: 1_000_000_000)
```

Comment:
```suggestion
                            try? await Task.sleep(for: .seconds(1))
```

See: https://developer.apple.com/documentation/swift/task/sleep(for:tolerance:clock:)

Reply:
Also, is there a specific reason, we need to wait for so long?

This looks like a workaround, is that right? If yes, can we document that here in a comment to explain for someone encountering this later? i.e. why we need to wait here, etc

Reply:
I agree with @pauljohanneskraft; I don't think we have to add the sleep in here as we will have the progress indicator in the message new. It will point out to the user that the questionnaire is processed and therefore we could already dismiss the questionnaire here and a user can no longer start the questionnaire again.